lib/Twig/Node.php

```diff
@@ -37,6 +37,11 @@ class Twig_Node implements Twig_NodeInterface
      */
     public function __construct(array $nodes = array(), array $attributes = array(), $lineno = 0, $tag = null)
     {
+        foreach ($nodes as $name => $node) {
+            if (!$node instanceof Twig_NodeInterface) {
```

Comment:
The ``getNode`` method already says it returns a ``Twig_Node``, not a ``Twig_NodeInterface``